lib/ansible/modules/cloud/amazon/ec2_vpc_net.py

```diff
@@ -396,6 +427,8 @@ def main():
             for cidr in to_add:
                 changed = True
                 connection.associate_vpc_cidr_block(CidrBlock=cidr, VpcId=vpc_id)
+        if ipv6_cidr:
+            connection.associate_vpc_cidr_block(AmazonProvidedIpv6CidrBlock=ipv6_cidr, VpcId=vpc_id)
```

Comment:
This and the `associate_vpc_cidr_block` call above should both be contained in a `try/except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError)`.